activesupport/lib/active_support/cache.rb

```diff
@@ -517,6 +539,17 @@ def merged_options(call_options)
           end
         end
 
+
```

Comment:
Let's nix that Rubocop rule then. I often use an extra return to separate logical units of related methods.